libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-roles.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.tabs.roles'">
+  <ng-container
+    *rxLet="
+      userRoleView$;
+      let userRoleView;
+      error: error;
+      suspense: suspense;
+      errorTrigger: hasGeneralError$
+    "
+  >
+    <watt-card>
+      <watt-card-title>
+        <h3>
+          {{ userRoleView | userRoleCount }}
```

Comment:
suggestion: instead of creating a pipe in this case, the `userRoleCount` can be a selector coming from the store. The logic related to calculating the count can be moved in the store as well. Similar to how `totalUserCount$` is implemented in `dh-users-tab.component.html`.

Reply:
Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1308/commits/f437d5794b7ede40cdc9c36c506d9452a5ec30ca